### PR TITLE
*: default value tiny update

### PR DESCRIPTION
**Pull request by zimulala**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @Frank945946

File: data-type-default-values.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 MySQL 从 8.0.13 开始支持在 `DEFAULT` 子句中指定表达式为默认值。具体可参考 [Explicit Default Handling as of MySQL 8.0.13](https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html#data-type-defaults-explicit)。
 
-TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 支持为 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
+TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 支持为 `BLOB`、`TEXT` 以及 `JSON` 数据类型（因这 3 个类型之前版本不支持，所以此处特别说明）分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```
Comment (Frank945946) [inline data-type-default-values.md:37]:
```suggestion
TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。其中 `BLOB`、`TEXT` 以及 `JSON` 这 3 种数据类型是从 v8.0.0 开始支持设置默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```

**Review by Frank945946 — APPROVED** [review]

File: data-type-default-values.md (hunk 1 above)
Comment (zimulala) [reply data-type-default-values.md:37]: 这个感觉和原来的描述差不多，还是会误解。

File: data-type-default-values.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这 3 个数据类型也可以设置默认值，但是默认值仅支持通过表达式来设置。[表达式默认值](#表达式默认值)。
```
Comment (tangenta) [inline data-type-default-values.md:11]:
```suggestion
从 v8.0.0 开始，包括 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三个数据类型在内的所有数据类型可以设置默认值，但是 `BLOB`、`TEXT` 以及 `JSON` 类型仅支持通过表达式来设置 [默认值](#表达式默认值)。
```

File: data-type-default-values.md (hunk 1 above)
Comment (tangenta) [inline data-type-default-values.md:37]:
```suggestion
TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 额外支持为 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```

File: data-type-default-values.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 MySQL 从 8.0.13 开始支持在 `DEFAULT` 子句中指定表达式为默认值。具体可参考 [Explicit Default Handling as of MySQL 8.0.13](https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html#data-type-defaults-explicit)。
 
-TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 支持为 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
+TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 额外支持 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```
Comment (tangenta) [inline data-type-default-values.md:37]:
```suggestion
TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 额外支持为 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```

**Review by tangenta — APPROVED** [review]

File: data-type-default-values.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+从 v8.0.0 开始，包括 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三个数据类型在内的所有数据类型都可以设置默认值，但是 `BLOB`、`TEXT` 以及 `JSON` 类型仅支持通过表达式来设置 [默认值](#表达式默认值)。
```
Comment (hfxsd) [inline data-type-default-values.md:11]:
```suggestion
从 v8.0.0 开始，所有数据类型都可以设置默认值，但是 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三个数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。
```

Comment (zimulala) [reply data-type-default-values.md:11]: 我感觉原来的好点，因为，比较突出那 3 中类型。v8.0 前的版本这 3 种类型是不支持的

**Review by hfxsd — APPROVED** [review]

File: data-type-default-values.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型。所有数据类型都可以设置默认值，但是 `BLOB`、`TEXT` 以及 `JSON`  这三个数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。
```
Comment (qiancai) [inline data-type-default-values.md:11]:
```suggestion
从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型支持通过表达式来设置[默认值](#表达式默认值)。在之前的版本中，这三种数据类型不支持设置默认值。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: data-type-default-values.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 MySQL 从 8.0.13 开始支持在 `DEFAULT` 子句中指定表达式为默认值。具体可参考 [Explicit Default Handling as of MySQL 8.0.13](https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html#data-type-defaults-explicit)。
 
-TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 支持为 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
+TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 额外支持为 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```
Comment (zimulala) [inline data-type-default-values.md:37]:
```suggestion
TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 额外支持 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```

File: data-type-default-values.md (hunk 5 above)
Comment (zimulala) [reply data-type-default-values.md:11]: 要不还是原来的的描述好一些？这个是前线不清楚这 3 个类型是新增支持，其它类型都支持。所以改成这样的。

Comment (qiancai) [reply data-type-default-values.md:11]: “从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型。” 原来这句话有点歧义，容易理解成这三种数据类型是 v8.0.0 新增支持的

Comment (zimulala) [inline data-type-default-values.md:11]:
```suggestion
从 v8.0.0 开始，额外支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置表达式默认值。所有数据类型都可以设置默认值，但是 `BLOB`、`TEXT` 以及 `JSON`  这三个数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。
```

File: data-type-default-values.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+从 v8.0.0 开始，额外支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置表达式默认值。所有数据类型都可以设置默认值，但是 `BLOB`、`TEXT` 以及 `JSON`  这三个数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。
```
Comment (zimulala) [inline data-type-default-values.md:11]:
```suggestion
从 v8.0.0 开始，额外支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置表达式默认值。即所有数据类型都可以设置默认值，只是 `BLOB`、`TEXT` 以及 `JSON`  这三个数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。
```

File: data-type-default-values.md [hunk 8]
```diff
@@ -8,7 +8,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+从 v8.0.0 开始，额外支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置表达式默认值。即所有数据类型都可以设置默认值，只是 `BLOB`、`TEXT` 以及 `JSON`  这三个数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。
```
Comment (hfxsd) [inline data-type-default-values.md:11]:
```suggestion
从 v8.0.0 开始，额外支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置表达式默认值。即所有数据类型都可以设置默认值，只是 `BLOB`、`TEXT` 以及 `JSON` 这三个数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。
```

File: data-type-default-values.md [hunk 9]
```diff
@@ -8,7 +8,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+从 v8.0.0 开始，额外支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置表达式默认值。即所有数据类型都可以设置默认值，只是 `BLOB`、`TEXT` 以及 `JSON` 这三个数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。
```
Comment (hfxsd) [inline data-type-default-values.md:11]:
```suggestion
从 v8.0.0 开始，额外支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置表达式默认值。即所有数据类型都可以设置默认值，只是 `BLOB`、`TEXT` 以及 `JSON` 这三种数据类型仅支持通过表达式来设置[默认值](#表达式默认值)。在 v8.0.0 之前，不支持设置这三种数据类型的默认值。
```

File: data-type-default-values.md [hunk 10]
```diff
@@ -8,7 +8,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置表达式默认值。即所有数据类型都可以设置默认值，只是 `BLOB`、`TEXT` 以及 `JSON` 这三种数据类型仅支持使用表达式来设置[默认值](#表达式默认值)。
```
Comment (hfxsd) [inline data-type-default-values.md:11]:
```suggestion
从 v8.0.0 开始，新增支持 [`BLOB`](/[data-type-string.md](http://data-type-string.md/)#blob-类型)、[`TEXT`](/[data-type-string.md](http://data-type-string.md/)#text-类型) 以及 [`JSON`](/[data-type-json.md](http://data-type-json.md/)#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```

File: data-type-default-values.md [hunk 11]
```diff
@@ -8,7 +8,7 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
 
 在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/[data-type-string.md](http://data-type-string.md/)#blob-类型)、[`TEXT`](/[data-type-string.md](http://data-type-string.md/)#text-类型) 以及 [`JSON`](/[data-type-json.md](http://data-type-json.md/)#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```
Comment (hfxsd) [inline data-type-default-values.md:11]:
```suggestion
所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```

**Pull request merged**